Sources/CombustionBLE/SimulatedProbe.swift

```diff
@@ -34,6 +34,7 @@ class SimulatedProbe: Probe {
         super.init(advertising, RSSI: SimulatedProbe.randomeRSSI(), identifier: UUID())
         
         firmareVersion = "v1.2.3"
+        hardwareRevision = "v2.3.4"
```

Comment:
should we use the real format for this e.g. "1.3-A1" since this is public?

Reply:
Good idea.  Fixed.